.idea/gradle.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This PR is unfit for review until all these unnecessary files are removed form the repo.